vips/vip-194/commands-testnet.ts

```diff
@@ -0,0 +1,39 @@
+const PRIME_LIQUIDITY_PROVIDER = "0xAdeddc73eAFCbed174e6C400165b111b0cb80B7E";
+
+const ETH = "0x98f7A83361F7Ac8765CcEBAB1425da6b341958a7";
+const BTC = "0xA808e341e8e723DC6BA0Bb5204Bafc2330d7B8e4";
+const USDC = "0x16227D60f7a0e586C66B005219dfc887D13C9531";
+const USDT = "0xA11c8D9DC9b66E209Ef60F0C8D969D3CD988782c";
+
+const VTreasury = "0x8b293600C50D6fbdc6Ed4251cc75ECe29880276f";
+
+export default [
+  {
+    target: PRIME_LIQUIDITY_PROVIDER,
+    signature: "setTokensDistributionSpeed(address[],uint256[])",
+    params: [
+      [ETH, BTC, USDC, USDT],
+      ["24438657407407", "1261574074074", "36881637731481481", "87191261574074074"],
```

Comment:
USDT and USDC have 6 decimals on testnet, so the right distributions speeds would be:
* USDC: 36881
* USDT: 87191